Tests/AnyLintTests/Checkers/TemplateCheckerTests.swift

```diff
@@ -0,0 +1,91 @@
+@testable import AnyLint
+@testable import Utility
+import XCTest
+
+final class TemplateCheckerTests: XCTestCase {
+    override func setUp() {
+        log = Logger(outputType: .test)
+        TestHelper.shared.reset()
+    }
+
+    func testPerformWithLocalSource() {
+        withTemporaryFiles(
+            [
+                (
+                    subpath: "AnyLint/Sample.swift",
+                    contents: """
+                        #!/usr/local/bin/swift-sh
+                        import AnyLint // @Flinesoft == wip/cg_template-system
+
+                        try Lint.reportResultsToFile(arguments: CommandLine.arguments) {
+                            // MARK: PseudoCheck
+                            try Lint.checkFilePaths(
+                                checkInfo: "PseudoCheck: Checks if the file `Pseudo.md` exists.",
+                                regex: #"^Pseudo\\.md$"#,
+                                matchingExamples: ["Pseudo.md"],
+                                nonMatchingExamples: ["Pseudo.markdown", "PSEUDO.md"],
+                                violateIfNoMatchesFound: true
+                            )
+                        }
+
+                        """
+                ),
+            ]
+        ) { filePaths in
+            let violations = try! TemplateChecker(source: .local(filePaths[0]), runOnly: nil, exclude: nil, logDebugLevel: false).performCheck()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Force tries should be avoided.](https://app.codacy.com/gh/Flinesoft/AnyLint/pullRequest?prid=5691372)